src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -80,6 +82,20 @@ public DocumentDatabase(string name, RavenConfiguration configuration, ServerSto
             {
                 serverStore?.DatabasesLandlord.UnloadResourceOnCatastrophicFailure(name, e);
             });
+            NonDurabaleFileSystemNotification = new NonDurabaleFileSystemNotification(e =>
+            {
+                if (serverStore != null)
+                {
+                    if (serverStore.NonDurableFileSystemNotified) // alert once per server (if at least on storage alerting)
```

Comment:
Hm... I wonder if we want to alert per _database_, instead?
The notification center should handle the unification of the alerts.